README.md

```diff
@@ -80,9 +80,12 @@ Now, you can run the pipeline using:
 
 ```bash
 nextflow run nf-core/rnaseq \
-    --input samplesheet.csv \
+    --input <SAMPLESHEET> \
     --outdir <OUTDIR> \
-    --genome GRCh37 \
+    --gtf <GTF> \
+    --fasta <GENOME FASTA> \
+    --igenomes_ignore \
+    --genome null \
```

Comment:
Why do we have to set these 2 as well?

Reply:
`--gtf <GTF> --fasta <FASTA>` should be enough

Reply:
pretty sure we added that (it's in usage.md) to fix an issue.

Reply:
https://github.com/nf-core/rnaseq/pull/1073/files#r1315129557

Reply:
Complains about what? If I test the following command on the current `dev` branch:

```
nextflow run . \
    --input https://raw.githubusercontent.com/nf-core/test-datasets/7f1614baeb0ddf66e60be78c3d9fa55440465ac8/samplesheet/v3.10/samplesheet_test.csv \
    --fasta https://raw.githubusercontent.com/nf-core/test-datasets/7f1614baeb0ddf66e60be78c3d9fa55440465ac8/reference/genome.fasta \
    --gtf https://raw.githubusercontent.com/nf-core/test-datasets/7f1614baeb0ddf66e60be78c3d9fa55440465ac8/reference/genes_with_empty_tid.gtf.gz \
    --outdir ./results \
    -profile docker
```

This is the output printed to stdout:

```
N E X T F L O W  ~  version 23.10.1
Launching `./main.nf` [suspicious_rutherford] DSL2 - revision: 5245a27840
WARN: Access to undefined parameter `monochromeLogs` -- Initialise it to a default value eg. `params.monochromeLogs = some_value`


------------------------------------------------------
                                        ,--./,-.
        ___     __   __   __   ___     /,-._.--~'
  |\ | |__  __ /  ` /  \ |__) |__         }  {
  | \| |       \__, \__/ |  \ |___     \`-._,-`-,
                                        `._,._,'
  nf-core/rnaseq v3.15.0dev
------------------------------------------------------
Core Nextflow options
  runName         : suspicious_rutherford
  containerEngine : docker
  launchDir       : /home/harshil/repos/nf-core/rnaseq
  workDir         : /home/harshil/repos/nf-core/rnaseq/work
  projectDir      : /home/harshil/repos/nf-core/rnaseq
  userName        : harshil
  profile         : docker
  configFiles     : 

Input/output options
  input           : https://raw.githubusercontent.com/nf-core/test-datasets/7f1614baeb0ddf66e60be78c3d9fa55440465ac8/samplesheet/v3.10/samplesheet_test.csv
  outdir          : ./results

Reference genome options
  fasta           : https://raw.githubusercontent.com/nf-core/test-datasets/7f1614baeb0ddf66e60be78c3d9fa55440465ac8/reference/genome.fasta
  gtf             : https://raw.githubusercontent.com/nf-core/test-datasets/7f1614baeb0ddf66e60be78c3d9fa55440465ac8/reference/genes_with_empty_tid.gtf.gz

Alignment options
  min_mapped_reads: 5

Max job request options
  max_memory      : 15.GB

!! Only displaying parameters that differ from the pipeline defaults !!
------------------------------------------------------
If you use nf-core/rnaseq for your analysis please cite:

* The pipeline
  https://doi.org/10.5281/zenodo.1400710

* The nf-core framework
  https://doi.org/10.1038/s41587-020-0439-x

* Software dependencies
  https://github.com/nf-core/rnaseq/blob/master/CITATIONS.md
------------------------------------------------------
WARN: Access to undefined parameter `gff` -- Initialise it to a default value eg. `params.gff = some_value`
WARN: Access to undefined parameter `transcript_fasta` -- Initialise it to a default value eg. `params.transcript_fasta = some_value`
WARN: Access to undefined parameter `additional_fasta` -- Initialise it to a default value eg. `params.additional_fasta = some_value`
WARN: Access to undefined parameter `salmon_index` -- Initialise it to a default value eg. `params.salmon_index = some_value`
[b6/ce61b9] process > NFCORE_RNASEQ:PREPARE_GENOME:GUNZIP_GTF (genes_with_empty_tid.gtf.gz)                        [100%] 1 of 1, cached: 1 ✔
[bc/d2e0a7] process > NFCORE_RNASEQ:PREPARE_GENOME:GTF_FILTER (genome.fasta)                                       [100%] 1 of 1, cached: 1 ✔
[f8/b7f79d] process > NFCORE_RNASEQ:PREPARE_GENOME:GTF2BED (genome.filtered.gtf)                                   [100%] 1 of 1 ✔
[55/32c851] process > NFCORE_RNASEQ:PREPARE_GENOME:MAKE_TRANSCRIPTS_FASTA (rsem/genome.fasta)                      [100%] 1 of 1 ✔
[8e/4489d2] process > NFCORE_RNASEQ:PREPARE_GENOME:CUSTOM_GETCHROMSIZES (genome.fasta)                             [100%] 1 of 1, cached: 1 ✔
[71/5903cd] process > NFCORE_RNASEQ:PREPARE_GENOME:STAR_GENOMEGENERATE (genome.fasta)                              [100%] 1 of 1 ✔
[93/7dfff0] process > NFCORE_RNASEQ:RNASEQ:CAT_FASTQ (WT_REP1)                                                     [100%] 2 of 2, cached: 2 ✔
[ec/21d7e3] process > NFCORE_RNASEQ:RNASEQ:FASTQ_FASTQC_UMITOOLS_TRIMGALORE:FASTQC (WT_REP1)                       [100%] 5 of 5, cached: 4 ✔
[ca/64ce5b] process > NFCORE_RNASEQ:RNASEQ:FASTQ_FASTQC_UMITOOLS_TRIMGALORE:TRIMGALORE (WT_REP1)                   [100%] 5 of 5, cached: 4 ✔

<TRUNCATED>
```

The `WARN` messages are expected here and something we need to fix before the next release.

Can someone try to run the same command on the current release please and report back here?

Reply:
I tested an it seems fine for me too. @vmkalbskopf if you can give us something reproducible that shows we need to retain this, we'll keep it, otherwise I'll simplify both here and in usage.md.

Reply:
Yeah, I tested it and I didn't run into the same errors as before. May have been differences in my command which referenced the iGenome option previously.